exec/cnex/cnex.c

```diff
@@ -842,19 +848,51 @@ void exec_INDEXAN(TExecutor *self)
     push(self->stack, val);
 }
 
-void exec_INDEXDR()
+void exec_INDEXDR(TExecutor *self)
 {
-    fatal_error("exec_INDEXDR not implemented");
+    self->ip++;
+    TString *index = string_fromString(top(self->stack)->string); pop(self->stack);
+    Cell *addr = top(self->stack)->address; pop(self->stack);
+    Cell *e = dictionary_findDictionaryEntry(addr->dictionary, index);
+    if (!e) {
```

Comment:
This is a pointer. Use an explicit `== NULL` check.

Reply:
Fixed.